src/pages/submit/linked-wearables.tsx

```diff
@@ -0,0 +1,429 @@
+  }),
+  motivation: (state) => ({
+    motivation: assert(state.motivation.length <= schema.motivation.maxLength, 'error.linked_wearables.motivation_too_large')
+  }),
+  method: (state) => ({
+    method: assert(state.method.length <= schema.method.maxLength, 'error.linked_wearables.method_too_large')
+  }),
+  '*': (state) => {
+    const links = Object.values(state.links)
+    const smart_contract = Object.values(state.smart_contract)
+    const managers = Object.values(state.managers)
+    return({
+      name: (assert(state.name.length > 0, 'error.linked_wearables.name_empty') ||
+      assert(state.name.length >= schema.name.minLength, 'error.linked_wearables.name_too_short') ||
+      assert(state.name.length <= schema.name.maxLength, 'error.linked_wearables.name_too_large')
+      ),
+      links: (
+        assert(links.some(option => option !== ''), `error.linked_wearables.smart_contract_empty`) ||
+        assert(links.every(option => isURL(option, {protocols: ['https']})), `error.linked_wearables.url_invalid`)
+      ),
+      introduction: (assert(state.introduction.length > 0, 'error.linked_wearables.introduction_empty') ||
+      assert(state.introduction.length >= schema.introduction.minLength, 'error.linked_wearables.introduction_too_short') ||
+      assert(state.introduction.length <= schema.introduction.maxLength, 'error.linked_wearables.introduction_too_large')
+      ),
+      nft_collections: (assert(state.nft_collections.length > 0, 'error.linked_wearables.nft_collections_empty') ||
+      assert(state.nft_collections.length >= schema.nft_collections.minLength, 'error.linked_wearables.nft_collections_too_short') ||
+      assert(state.nft_collections.length <= schema.nft_collections.maxLength, 'error.linked_wearables.nft_collections_too_large')
+      ),
+      smart_contract: (
+        assert(smart_contract.some(option => option !== ''), `error.linked_wearables.smart_contract_empty`) ||
+        assert(smart_contract.every(option => isEthereumAddress(option)), `error.linked_wearables.address_invalid`)
+      ),
+      governance: (assert(state.governance.length > 0, 'error.linked_wearables.governance_empty') ||
+      assert(state.governance.length >= schema.governance.minLength, 'error.linked_wearables.governance_too_short') ||
+      assert(state.governance.length <= schema.governance.maxLength, 'error.linked_wearables.governance_too_large')
+      ),
+      motivation: (assert(state.motivation.length > 0, 'error.linked_wearables.motivation_empty') ||
+      assert(state.motivation.length >= schema.motivation.minLength, 'error.linked_wearables.motivation_too_short') ||
+      assert(state.motivation.length <= schema.motivation.maxLength, 'error.linked_wearables.motivation_too_large')
+      ),
+      method: (assert(state.method.length <= schema.method.maxLength, 'error.linked_wearables.method_too_large')
```

Comment:
I think we are missing the method too short validation

Reply:
It's missing because it's optional 🤔